packages/shared/src/components/typography/Typography.tsx

```diff
@@ -62,6 +63,7 @@ export function Typography<TagName extends AllowedTags>({
   className,
   ...props
 }: TypographyProps<TagName>): ReactElement {
-  const Tag = classed(tag, className, type, bold && 'font-bold', color);
+  const classes = classNames(className, type, bold && 'font-bold', color);
+  const Tag = classed(tag, classes);
```

Comment:
There's an issue with the `classed` component when the first parameter `className` is null or undefined. If that is the case, the following classes are not being applied.

Reply:
wow good find!
CC: @capJavert just so you know, maybe this was the case 🤯

Reply:
Didn't want to touch the `classed` file yet, but we can maybe write a ticket for it if anyone wants 😅